src/04-modality-specific-files/05-task-events.md

```diff
@@ -129,16 +197,17 @@ of example.
    "identifier": {
        "LongName": "Unique identifier from Karolinska (KDEF) database",
        "Description": "ID from KDEF database used to identify the displayed image"
-   }
+   },
+    "StimulusPresentation": {
+        "OperatingSystem": "Linux-Ubuntu 18.04.5 LTS",
+        "SoftwareName": "Psychtoolbox",
+        "SoftwareRRID": "SCR_002881",
+        "SoftwareVersion": "3.0.14",
+        "Code": "doi:10.5281/zenodo.3581316"
```

Comment:
```suggestion
        "Code": "doi:10.5281/zenodo.3361717"
```

Not sure how but this DOI was still pointing to my very messy code rather than @sappelhoff much cleaner example.

Also I am realizing the details of this "synthethic" example do not match the content of the example code: 
previous DOI pointed to Presentation code, new example is PsychoPy (?)  code. Neither used the Karolinska database.

This could confuse people.

Reply:
Will add this new DOI for now as it points to a better example of code anyway.

But I think we might want to reflect on whether our json examples that point to "real" things have to be internally consistent.